.github/workflow/ci.yaml

```diff
@@ -0,0 +1,28 @@
+name: CI pipeline
+
+on:
+  push:
+    branches:
+      - "**" # matches every branch
+      
+jobs:
+  lint:
+    name: checkout
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+
+      - uses: actions/setup-python@v2
```

Comment:
lets try the latest version of this action here https://github.com/actions/setup-python